doc/nrf/device_guides/working_with_nrf/nrf54l/peripheral_sensor_node_shield.rst

```diff
@@ -0,0 +1,88 @@
+.. _peripheral_sensor_node_shield:
+
+Peripheral Sensor node Shield
+#############################
+
+.. contents::
+   :local:
+   :depth: 2
+
+Overview
+********
+
+The Peripheral Sensor node shield is an extension for the :ref:`nRF54L15 PDK <ug_nrf54l15_gs>`.
+It is a testing shield with following sensors:
+
+   * ``ADXL362`` - Accelerometer (SPI interface)
+   * ``BMI270`` - Accelerometer and gyroscope (SPI interface)
+   * ``BME688`` - Temperature, pressure, humidity and gas sensor (I2C interface with on-board pull-up resistors 4.7 kohm).
+                  It is targeted for testing I2C, and SPI protocols drivers.
```

Comment:
"It is targeted for..." is a general note related to the shield. What about removing spaces from the beginning of that line?

Reply:
Based on other documentation, it seems that this alignment should remain.
And Compliance test is green.